### PR TITLE
SEQNG-887: Final touches on SEQNG-887

### PR DESCRIPTION
**Pull request by cquiroz**
I've been doing this large refactoring to make all epics reads referentially transparent. While some improvements can still be done we should close this

This PR removes some duplicated code that was able to work on the old and new world and rename `EpicsCommandF` to `EpicsCommand` and `ObsecveCommandF` to `ObserveCommand`

As changes go this PR is fairly mechanical

### CODE REVIEW

File: modules/seqexec/server/src/main/scala/seqexec/server/EpicsUtil.scala [hunk 1]
```diff
@@ -124,33 +90,18 @@ trait EpicsSystem[T] {
...
-      p.map(_.set(f(v)).asRight).getOrElse(SeqexecFailure.Unexpected("Unable to set parameter.").asLeft)
-    } )
-
-  def setParameter[T](p: Option[CaParameter[T]], v: T): SeqAction[Unit] =
-    safe(SeqAction.either {
-      p.map(_.set(v).asRight).getOrElse(SeqexecFailure.Unexpected("Unable to set parameter.").asLeft)
-    } )
+  sealed trait Result extends Product with Serializable
+  case object Paused extends Result
+  case object Completed extends Result
```
Comment (tpolecat) [inline modules/seqexec/server/src/main/scala/seqexec/server/EpicsUtil.scala]: I think elsewhere we're using a style where instances are defined inside the companion, so `Result.Paused`, etc.

File: modules/seqexec/server/src/main/scala/seqexec/server/EpicsUtil.scala [hunk 2]
```diff
@@ -262,6 +179,10 @@ object EpicsCodex {
 }
 
 object EpicsUtil {
+  def safe[A](a: SeqAction[A]): SeqAction[A] = EitherT(a.value.attempt.map {
```
Comment (tpolecat) [inline modules/seqexec/server/src/main/scala/seqexec/server/EpicsUtil.scala]: Can you add a comment explaining what this does?

File: modules/seqexec/server/src/main/scala/seqexec/server/EpicsUtil.scala (hunk 1 above)
Comment (cquiroz) [reply modules/seqexec/server/src/main/scala/seqexec/server/EpicsUtil.scala]: yes, I have been changing some but I missed this one

File: modules/seqexec/server/src/main/scala/seqexec/server/EpicsUtil.scala [hunk 3]
```diff
@@ -159,41 +110,7 @@ object EpicsCommand {
...
-
-  def mark: SeqAction[Unit] = safe(EitherT(IO.apply {
-    cs.map(_.mark().asRight).getOrElse(SeqexecFailure.Unexpected("Unable to mark command.").asLeft)
-  }))
-
-  def setTimeout(t: Time): SeqAction[Unit] = EpicsUtil.setTimeout(cs.map(_.getApplySender), t)
-}
-
-trait ObserveCommandF {
+trait ObserveCommand     {
```
Comment (jluhrs) [inline modules/seqexec/server/src/main/scala/seqexec/server/EpicsUtil.scala]: Why that much space before the `{` ?

Comment (cquiroz) [reply modules/seqexec/server/src/main/scala/seqexec/server/EpicsUtil.scala]: just a mistake

**Pull request merged**